flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java

```diff
@@ -103,6 +103,18 @@ public void putProperties(DescriptorProperties otherProperties) {
 		}
 	}
 
+	/**
+	 * Adds a properties map by appending the given prefix to element keys.
+	 */
+	public void putPropertiesWithPrefix(String prefix, Map<String, String> prop) {
```

Comment:
Maybe the '.' should be part of the prefix rather than being hardcoded here. Doing this should simplify both doc and code.

Reply:
'.' is widely used in the descriptor. I'll add an example to the doc

Reply:
'.' is widely used in this class. I'll add an example to the doc to make it clearer